python/utils.py

```diff
@@ -279,16 +279,24 @@ def override_build_config_from_env(config):
 
 
 class build_ext(setuptools.command.build_ext.build_ext):
+    def __init__(self, *args, **kwargs):
+        self.install_dir = kwargs.pop("install_dir", None)
+        super().__init__(*args, **kwargs)
+
     def build_extension(self, ext):
         if ext.name == "nvfuser._C":
             # Copy files on necessity.
             filename = self.get_ext_filename(self.get_ext_fullname(ext.name))
             fileext = os.path.splitext(filename)[1]
 
             libnvfuser_path = os.path.join(
-                "./nvfuser_common/lib", f"libnvfuser{fileext}"
+                os.path.join(self.install_dir, "lib"), f"libnvfuser{fileext}"
             )
-            assert os.path.exists(libnvfuser_path)
+            if not os.path.exists(os.path.dirname(libnvfuser_path)):
+                os.makedirs(os.path.dirname(libnvfuser_path))
+            if not os.path.exists(libnvfuser_path):
```

Comment:
Not sure why you'd create an empty file.
It is supposed to copy from `install_dir/lib/libnfuser.so` to `python/nvfuser/_C.cpython-312-aarch64-linux-gnu.so`.
It would be an error if the shared library is not found.